resource_booking/__manifest__.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2021 Tecnativa - Jairo Llopis
+# Copyright 2022 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Resource booking",
+    "summary": "Manage appointments and resource booking",
+    "version": "16.0.1.0.0",
+    "development_status": "Production/Stable",
+    "category": "Appointments",
+    "website": "https://github.com/OCA/calendar",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "maintainers": ["pedrobaeza", "ows-cloud"],
+    "license": "AGPL-3",
+    "application": True,
+    "installable": True,
+    "external_dependencies": {
+        "python": [
+            # Used implicitly
+            "cssselect",
+            "freezegun",
```

Comment:
This is not correct. This dependency is not for the module, but for tests. You must remove it from here and put it in `test_requirements.txt` file in root.

Reply:
@norlinhenrik I have pushed to https://github.com/ows-cloud/calendar/pull/1 a commit fixing this

Reply:
`cssselect` may be needed for the module itself. I was talking about `freezegun`. Anyway, the CI is still red.